src/test/java/com/cdancy/bitbucket/rest/features/PullRequestApiMockTest.java

```diff
@@ -369,4 +373,141 @@ public void testGetPullRequestCommitsOnError() throws Exception {
+        BitbucketApi baseApi = api(server.getUrl("/"));
+        PullRequestApi api = baseApi.pullRequestApi();
+        try {
+
+            ParticipantsPage participants = api.listParticipants(project, repo, 1, 5, 0);
+            assertThat(participants).isNotNull();
+            assertThat(participants.errors()).isEmpty();
+            assertThat(participants.values()).hasSize(1);
+            assertThat(participants.values().get(0).approved()).isFalse();
+
+            Map<String, ?> queryParams = ImmutableMap.of("start", "0", "limit", 5);
+            assertSent(server, "GET", "/rest/api/" + BitbucketApiMetadata.API_VERSION
+                    + "/projects/PRJ/repos/my-repo/pull-requests/1/participants", queryParams);
+        } finally {
+            baseApi.close();
+            server.shutdown();
+        }
+    }
+
+    public void testGetPullRequestParticipantsOnError() throws Exception {
+        MockWebServer server = mockEtcdJavaWebServer();
+
+        server.enqueue(new MockResponse().setBody(payloadFromResource("/pull-request-participants-error.json"))
+                .setResponseCode(200));
+        BitbucketApi baseApi = api(server.getUrl("/"));
+        PullRequestApi api = baseApi.pullRequestApi();
+        try {
+
+            ParticipantsPage participants = api.listParticipants(project, repo, 1, 5, 0);
+            assertThat(participants).isNotNull();
+            assertThat(participants.errors()).isNotEmpty();
+
+            Map<String, ?> queryParams = ImmutableMap.of("start", "0", "limit", 5);
+            assertSent(server, "GET", "/rest/api/" + BitbucketApiMetadata.API_VERSION
+                    + "/projects/PRJ/repos/my-repo/pull-requests/1/participants", queryParams);
+        } finally {
+            baseApi.close();
+            server.shutdown();
+        }
+    }
+
+    public void testUpdatePullRequestPaticipants() throws Exception {
+        MockWebServer server = mockEtcdJavaWebServer();
+
+        server.enqueue(new MockResponse().setResponseCode(204));
+        BitbucketApi baseApi = api(server.getUrl("/"));
+        PullRequestApi api = baseApi.pullRequestApi();
+        try {
+            String projectKey = "PRJ";
+            String repoKey = "myrepo";
+            Long pullRequestId = 839L;
+            User user = User.create("bob", "bob@acme.ic", 123, "bob", true, "bob", "asd");
+            Participants participants = Participants.create(user, null, Participants.Role.REVIEWER, false, Participants.Status.UNAPPROVED);
+            boolean success = api.updateParticipants(projectKey, repoKey, pullRequestId, participants);
+            assertThat(success).isTrue();
+            assertSent(server, "POST", "/rest/api/" + BitbucketApiMetadata.API_VERSION
+                    + "/projects/" + projectKey + "/repos/" + repoKey + "/pull-requests/"
+                    + pullRequestId + "/participants");
+        } finally {
+            baseApi.close();
+            server.shutdown();
+        }
+    }
```

Comment:
Lets ensure we have its opposing test case `testUpdatePullRequestPaticipantsOnError`.